config/config.go

```diff
@@ -25,13 +26,16 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-func LoadFile(paths []string, expandEnvVars bool) (*Config, error) {
+func LoadFile(logger *slog.Logger, paths []string, expandEnvVars bool) (*Config, error) {
 	cfg := &Config{}
 	for _, p := range paths {
 		files, err := filepath.Glob(p)
 		if err != nil {
 			return nil, err
 		}
+		if len(files) == 0 && logger != nil {
+			logger.Warn("No file found matching pattern", "file", p)
```

Comment:
```suggestion
			logger.Warn("No file found matching pattern", "path", p)
```

Reply:
The command line argument is `--config.file`, so I thought "file" was less confusing, but happy either way

Reply:
Hmm, true. `file` is fine then.